src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -574,10 +574,16 @@ private function registerAssetsConfiguration(array $config, ContainerBuilder $co
     {
         $loader->load('assets.xml');
 
+        $removeGitStrategy = true;
```

Comment:
> Btw, I think the git process should have its working directory forced to kernel.root_dir.

It is already implemented:
```$process = new Process('git rev-parse HEAD', $container->getParameter('kernel.root_dir'));```

Ok, I will do another changes. But actually don't see why it is better than current implementation, just will be in one place, but not so flexible on my mind.